tasks/kubectl-run-regression-acceptance-tests.yml

```diff
@@ -0,0 +1,42 @@
+platform: linux
```

Comment:
Could tags be passed into the job as a param, so we only need one task definition?
🎨